android/app/src/main/java/org/fossasia/openevent/utils/CustomTabIntent.java

```diff
@@ -0,0 +1,51 @@
+package org.fossasia.openevent.utils;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.content.pm.ResolveInfo;
+import android.graphics.BitmapFactory;
+import android.net.Uri;
+import android.support.customtabs.CustomTabsIntent;
+import android.support.v4.content.ContextCompat;
+
+import org.fossasia.openevent.R;
+
+import java.util.List;
+
+public class CustomTabIntent {
+    private Activity context;
+
+    public CustomTabIntent(Activity context) {
+        this.context = context;
+    }
+
+    public void launchUrl(String url) {
+
+        boolean customTabsSupported;
+        Intent customTabIntent = new Intent("android.support.customtabs.action.CustomTabsService");
+        customTabIntent.setPackage("com.android.chrome");
+        List<ResolveInfo> resolveInfos = context.getPackageManager().queryIntentServices(customTabIntent, 0);
+        customTabsSupported = !(resolveInfos == null || resolveInfos.isEmpty());
+
+        String URL = url;
+        if (!url.startsWith("http://") && !url.startsWith("https://")) {
+            URL = ("http://" + url);
+        }
+
+
+        if (customTabsSupported) {
```

Comment:
I think CustomTabs have backword compabilities it will authomatically start intent to open URL if custom tabs is not supported.

Reply:
I have tested it on API 18

Reply:
I will remove this part then. Thanks for telling me this.